config/forms/2021_2022.json

```diff
@@ -3,11 +3,11 @@
   "start_year": 2021,
   "end_year": 2022,
   "sections": {
-    "about_this_log": {
-      "label": "About this log",
+    "setup": {
+      "label": "Before you start",
       "subsections": {
-        "about_this_log": {
-          "label": "About this log",
+        "setup": {
+          "label": "Set up your lettings log",
```

Comment:
Given the form has `"type": "lettings"` I wouldn’t have thought this would be a problem; sales would be in a different definition.

Reply:
Think that was an initial idea but that key doesn't actually do anything right now (should maybe be removed). Possible that we put sales in it's own definition file - but I think you still have the problem of how do you know which this is at this first point since we only ask if it's a sale or letting half way through About this Log?

Reply:
Given all the questions are lettings related for now, and we will launch private beta only with lettings, think this is fine. Also, in the prototype we’re looking to ask if want to create a sales or lettings before setting up the new log, partly because the form groupings and labels are so radical different (cf. references to ‘tenancy).

Which makes me think… maybe we should remove the Lettings/Sale question in this JSON form?

<img width="515" alt="Screenshot 2022-01-27 at 15 55 13" src="https://user-images.githubusercontent.com/813383/151394846-7e597e76-5920-4c92-866d-034a435d3a53.png">

Reply:
Ok I think that's fine if that's the approach we're taking but then we should probably also remove the sales/lettings questions as you suggested in this PR.

Reply:
Have removed the sales related question and its dependencies.